software_steering_council.md

```diff
@@ -0,0 +1,52 @@
+# Software Steering Council
+
+## Purpose
+
+The Jupyter Software Steering Council (SSC) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy.
+
+## Responsibilities
+
+### Sole responsibilities
+
+The SSC is responsible for the following:
+
+- Defining the submission, review, and approval process for Jupyter Enhancement Proposals.
+- Stewarding the JEP process to ensure that it is inclusive and participatory, and solicits feedback from the right stakeholders. As a starting point, [the NumFOCUS DISCOVER Cookbook](https://github.com/numfocus/DISCOVER-Cookbook) may offer some useful starting pointers.
+- Making decisions on JEPs after receiving community feedback (JEPs are the mechanism for elevating any software decision/idea up to the broader Project, while individual Subprojects can make decisions within their scope via their regular mechanisms).
+- Owning, managing and implementing the Jupyter Incubation process (https://github.com/jupyter-incubator).
+- Owning, managing and implementing the Jupyter Attic process (https://github.com/jupyter-attic).
+- Stewarding discussions and activities beyond JEPs that involve cross-cutting concerns, standards, protocols and other architectural issues that impact the entire project.
+- Owning and managing anything related to security vulnerabilities across the project, including the Jupyter security lists, any private security repositories, etc.
+
+### Shared responsibilities with the Board of Directors
+
+The SSC and BoD share the responsibility for:
+
+- Updates or amendments to the [Jupyter Code of Conduct](https://jupyter.org/governance/conduct/code_of_conduct.html).
```

Comment:
I disagree. I believe the BoD should be responsible for this and the SSC should be responsible along with the BoD that the CoC is upheld in the subprojects.

Further, there should be a Diversity and Inclusion Subproject on the SSC as well as a Diversity and Inclusion standing workgroup of the BoD. As it stands now, this governance structure effectively eliminates gender inclusion. The progress of inclusion on many subprojects has been poor. While we can talk about how important inclusion is to the project, the reality is that the actions of the project have not made it a priority. Creating a governance is the place to bake in inclusion by default. If we only hope it will organically happen, it will not (look at the past five years as evidence).

Reply:
Thanks @willingc for these - sorry we didn't get last week to this one, but we're going in order :)

> I disagree. I believe the BoD should be responsible for this and the SSC should be responsible along with the BoD that the CoC is upheld in the subprojects.

We discussed this on Friday and I think those of us who were there (Steve, Sharan, Darian, me) lean in the same direction as you now... I think this has been part of the back-and-forth on how to best apply divisions of labor/scope between the SSC and the BoD, but it probably makes sense to leave the CoC definition/updates to the BoD. Obviously updates to that document will still be subject to community input/feedback, so there's no loss of access to that discussion for SSC members. But it does help clarify that the SSC's scope is software architecture, while the BoD focuses on community and governance.

> Further, there should be a Diversity and Inclusion Subproject on the SSC as well as a Diversity and Inclusion standing workgroup of the BoD. As it stands now, this governance structure effectively eliminates gender inclusion. The progress of inclusion on many subprojects has been poor. While we can talk about how important inclusion is to the project, the reality is that the actions of the project have not made it a priority. Creating a governance is the place to bake in inclusion by default. If we only hope it will organically happen, it will not (look at the past five years as evidence).

The way we'd envisioned topics like D&I was that the working groups on such matters (much like events, trademarks, etc.) ought to be chartered by the BoD, and then for those whose scope would intersect with the SSC, then they would jointly agree to have a representative from that working group in the SSC. So rather than having two separate diversity subprojects (at least that's how I understood your paragraph above), there should be a single one targeting these issues across the entire Project Jupyter, with a representative in the SSC who can focus on Diversity questions in the process of software.

Reply:
We've updated the document as per this feedback, and will also jointly update the BoD draft (PR upcoming) to match. Thanks!